app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala

```diff
@@ -165,7 +165,8 @@ class MovementsRepositoryImpl @Inject() (
         Codecs.playFormatCodec(mongoFormats.paginationMovementSummaryFormat),
         Codecs.playFormatCodec(mongoFormats.paginationMessageSummaryFormat),
         Codecs.playFormatCodec(mongoFormats.sensitiveStringFormat)
-      )
+      ),
+      replaceIndexes = appConfig.replaceIndexes
```

Comment:
is this required?

Reply:
Yes, If we have an existing collection then for the indexes to take effect we need to enforce it.
Jack said we need to trigger this and also there's a slack thread in community-mongo also indicating that we need to do this.